src/uct/ib/ud/base/ud_iface.c

```diff
@@ -356,23 +354,10 @@ ucs_status_t uct_ud_iface_complete_init(uct_ud_iface_t *iface)
         goto err_twheel_cleanup;
     }
 
-    status = ucs_async_set_event_handler(async_mode, iface->super.comp_channel->fd,
-                                         POLLIN, uct_ud_iface_event, iface, async);
-    if (status != UCS_OK) {
-        goto err_remove_timer;
-    }
-
-    status = uct_ib_iface_arm_rx_cq(&iface->super, 1);
-    if (status != UCS_OK) {
-        goto err_unset_event_handler;
-    }
-
+    uct_base_iface_progress_enable(&iface->super.super.super,
```

Comment:
Wow... Is there any other kind of async progress? How about async completions/ACKs? I'm asking because it can cause false positive peer failure for UD if app does not call progress long time.